### ISSUE TITLE
Accessibility: User should be able to start game with Enter

### ISSUE THREAD

**Issue by Hanna-Holm** [Hanna-Holm/u10-business-idea#4]
- User should be able to start the game by pressing enter after entering name as well as pressing the start button with the mouse
